src/components/card.vue

```diff
@@ -0,0 +1,123 @@
+<template>
+  <div :class="['sora-card', isActive ? 'active' : '']" >
+    <div class="card_icon">
+      <img :src="require(`@/assets/icons/${icon}.svg`)" />
+    </div>
+    <div class="card_desc">
+      <span class="card_desc-title">TOTAL {{ token.toUpperCase() }} SUPPLY</span>
+      <span class="card_desc-amount">{{ supply.v }}.{{ supply.dp }}</span>
+    </div>
+    <div class="card_overlay" :style="overlayStyle" />
+  </div>
+</template>
+
+<script>
+import { TOKENS } from '@/consts'
+
+export default {
+  name: 'sora-card',
+  props: {
+    currentToken: {
+      type: String,
+      required: true,
+      default: ''
+    },
+    token: {
+      type: String,
+      required: true,
+      default: ''
+    },
+    supply: {
+      type: Object,
+      required: true,
+      default: () => ({
+        v: '0',
+        dp: '0'
+      })
+    }
+  },
+  computed: {
+    isActive () {
+      return this.token === this.currentToken
+    },
+    icon () {
```

Comment:
Is it the same as `icon () { return this.token }`? Or we can have not only XOR and VAL?